### PR TITLE
Improve legibility of admin pages by indenting JSON.

### PR DESCRIPTION
**Pull request by nt**
Improve legibility of admin pages by indenting JSON.

### CODE REVIEW

**Comment by nt** [comment]
cc @htuch @alyssawilk

File: source/common/protobuf/utility.cc [hunk 1]
```diff
@@ -64,6 +64,7 @@ std::string MessageUtil::getJsonStringFromMessage(const Protobuf::Message& messa
   // Setting this option makes debugging easier because it keeps field names consistent in JSON
   // printouts.
   json_options.preserve_proto_field_names = true;
+  json_options.add_whitespace = true;
```
Comment (htuch) [inline source/common/protobuf/utility.cc]: This is used in various wire serializations. I think we probably don't want extra whitespace there universally. How about making this a bool arg that defaults false, and you can opt in stuff like admin pages?

**Comment by dnoe** [comment]
I don't see the DCO signed-off-by, but maybe I'm missing something. It'll need to be in your commit message.

**Comment by nt** [comment]
Sounds good, will send an updated version.

**Comment by mattklein123** [comment]
FYI this is being done already in https://github.com/envoyproxy/envoy/pull/2449. cc @ramaraochavali

**Comment by dnoe** [comment]
@nt does this do anything special that isn't covered under #2449 ?

**Comment by nt** [comment]
This is covered in #2449, thanks for bringing it to my attention; closing this PR.

**Pull request closed without merging**